cli_tools/common/utils/logging/service/logger.go

```diff
@@ -237,6 +237,12 @@ func (l *Logger) runWithServerLogging(function func() (*daisy.Workflow, error),
 	return logExtension, err
 }
 
+func removeNewLinesFromMultilineError(s string) string {
+	// first line in a multi error line is of "Multiple errors" type and doesn't need a separator
+	firstNewLineRemoved := strings.Replace(s, "\n", " ", 1)
+	return strings.Replace(firstNewLineRemoved, "\n", "; ", -1)
```

Comment:
minor: Any reason to not use `ReplaceAll`? (I had to read the source code of strings.Replace to figure out what `-1` meant.)